doc/specs/stdlib_linalg.md

```diff
@@ -168,3 +168,41 @@ program demo_trace
     print *, trace(A) ! 1 + 5 + 9
 end program demo_trace
 ```
+
+## `outer_product` - Computes the outer product of two vectors
+
+### Status
+
+Experimental
+
+### Description
+
+Computes the outer product of two vectors
```

Comment:
I suggest to swap the sections `Description` and `Status`

Reply:
Thank you for the comments.
This one puzzles me though, partly because I'm not sure exactly what you are suggesting. Are you suggesting we swap the contents of these sections, the order of these two sections, or something else?
In any case, it's not clear to me why we would want to do this. Thanks.

Reply:
Sorry for the confusion. What I tried to suggest is to have the following order:
```
### Description

Reply:
>  Perhaps if we decide to switch the order for these sections in the docs we would want to make a separate issue and pull rather than tucking it away in this PR? If I were to change just my function it wouldn't match the rest of the docs here.

I agree with you. Sorry, and ignore this comment.